docs-public/snippets.js

```diff
@@ -110,23 +111,24 @@ var plunkerTemplate = kendo.template(
         .event-log li:last-child { margin-bottom: -1px;}\
     </style>\
     <script src="https://unpkg.com/core-js/client/shim.min.js"></script>\
+    # if (data.platform === \'builder\') { # <script src="https://unpkg.com/zone.js"></script> #}#\
```

Comment:
Using a specific zone.js version will improve performance, because less requests to unpkg are made, and may prevent accidental errors (when a new major version is released).